test/integration/parallel_worker_test.js

```diff
@@ -15,12 +15,11 @@ suite('Parallel workers', function() {
   var PARALLEL_TOTAL = 2;
 
   var workerA, workerB;
-  var workerType = `test_worker_${slugid.v4()}`.substring(0,22);
 
   setup(co(function * () {
     // Each worker should use the same worker type but a unique worker id.
-    workerA = new TestWorker(DockerWorker, workerType, slugid.v4());
-    workerB = new TestWorker(DockerWorker, workerType, slugid.v4());
+    workerA = new TestWorker(DockerWorker, 'dummy-parallel-type');
+    workerB = new TestWorker(DockerWorker, 'dummy-parallel-type');
```

Comment:
By making these names not unique how confident are you that we can still run these tests in parallel (i.e. your computer/my computer) or ideally eventually parallel by default on a single machine